runtime/interpreter/primitivestatictype.go

```diff
@@ -39,6 +43,18 @@ func (t PrimitiveStaticType) Equal(other StaticType) bool {
 	return t == otherPrimitiveType
 }
 
+func (t PrimitiveStaticType) MeteredString(memoryGauge common.MemoryGauge) string {
+	if str, ok := PrimitiveStaticTypes[t]; ok {
+		common.UseMemory(memoryGauge, common.NewRawStringMemoryUsage(len(str)))
+		return str
+	}
+
+	l := 21 + OverEstimateIntStringLength(int(t))
+	common.UseMemory(memoryGauge, common.NewRawStringMemoryUsage(l))
+
+	return "PrimitiveStaticType(" + strconv.FormatInt(int64(t), 10) + ")"
```

Comment:
Maybe calculate the magic number from the constant part of the formatted string, e.g. 
```go
const primitiveStaticTypePrefix = "PrimitiveStaticType"
// + 2 for parentheses
var primitiveStaticTypeConstantLength = len(primitiveStaticTypePrefix) + 2
```
and then `return fmt.Sprintf("%s(%d)", primitiveStaticTypePrefix, t)`

Reply:
Added in 49b9ebf